auth_totp/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 {
     'name': 'MFA Support',
     'summary': 'Allows users to enable MFA and add optional trusted devices',
-    'version': '10.0.1.0.0',
+    'version': '10.0.1.0.1',
```

Comment:
not to be picky but I think it should be 10.0.1.1.0 as it requires a module update.